icu_benchmarks/data/split_process_data.py

```
@@ -1,12 +1,13 @@
 import copy
 import logging
 import os
-
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)